### PR TITLE
ME-14565-subtitles-styles

### PR DESCRIPTION
**Pull request by tsi**

### CODE REVIEW

File: src/plugins/styled-text-tracks/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default async function lazyPlugin(config) {
+  const player = this;
+  try {
+    const { default: initPlugin } = await import('./styled-text-tracks');
```
Comment (ehab-cl) [inline src/plugins/styled-text-tracks/index.js:4]: 💪

File: src/plugins/styled-text-tracks/styled-text-tracks.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
+.cld-video-player {
+  // Positions
+  // Center
```
Comment (ehab-cl) [inline src/plugins/styled-text-tracks/styled-text-tracks.scss:3]: I would say the positions are pretty clear from the class names (if you agree with me applies to the following classes as well, including themes)
```suggestion
```

**Review by ehab-cl — APPROVED** [review]
LGTM 💪

File: src/plugins/styled-text-tracks/styled-text-tracks.scss (hunk 2 above)
Comment (tsi) [reply src/plugins/styled-text-tracks/styled-text-tracks.scss]: Yeah, I was using them as more of separators rather than to clarify meaning. Removed.

**Review by ehab-cl — APPROVED** [review]

**Pull request merged**